src/systems/systems/src/hydraulic/brake_circuit.rs

```diff
@@ -263,7 +318,11 @@ impl BrakeCircuit {
     }
 
     pub fn accumulator_fluid_volume(&self) -> Volume {
-        self.accumulator.fluid_volume()
+        if let Some(accumulator) = &self.accumulator {
+            accumulator.fluid_volume()
+        } else {
+            Volume::default()
+        }
```

Comment:
Could write it like this:
```suggestion
        self.accumulator
            .as_ref()
            .map_or(Volume::default(), |acc| acc.fluid_volume())
```
or like this:
```suggestion
        self.accumulator
            .as_ref()
            .map_or(Volume::default(), Accumulator::fluid_volume)
```